zimui/index.html

```diff
@@ -12,6 +12,9 @@
   </head>
   <body>
     <div id="app"></div>
+    <noscript>
+      <p>JavaScript is disabled in your browser. Please enable JavaScript to view this ZIM.</p>
```

Comment:
```suggestion
      <p>JavaScript is disabled in your browser. Please enable JavaScript to access this content.</p>
```

WDYT?

Reply:
I'll use a middleground: `Please enable JavaScript to access content inside this ZIM.`